topics/admin/tutorials/ansible-galaxy/tutorial.md

```diff
@@ -423,12 +448,28 @@ The configuration is quite simple thanks to the many sensible defaults that are
 >    > >     database_connection: "postgresql:///galaxy?host=/var/run/postgresql"
 >    > >     file_path: /data
 >    > >     check_migrate_tools: false
+>    > >     shed_tool_data_dir: "{{ galaxy_mutable_data_dir }}/tool-data"
 >    > > ```
 >    > > {% endraw %}
 >    > {: .solution }
 >    >
 >    {: .question}
 >
+>    > ### {% icon comment %} Ansible Variable Templating
```

Comment:
Maybe we could introduce this in the general Ansible tutorial and reference it here? Not blocking this PR, just a thought.

Reply:
It's a good thought, mind if I include in both places? Since we don't have multiple roles run during the Ansible training, it's more of an "interesting note" there, but here it's quite relevant and there's a very real example.

Reply:
Sure!